models/rover_ackermann/model.config

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0"?>
+<model>
+  <name>rover_ackermann</name>
+  <version>1.0</version>
+  <sdf version="1.8">model.sdf</sdf>
+  <author>
+    <name>Benjamin Perseghetti</name>
+    <email>bperseghetti@rudislabs.com</email>
+  </author>
+  <description>A model of the NXP Mobile Robotics Buggy3.</description>
```

Comment:
Where is this model copied from? Please confirm that the license for this model permits us to have this model under our repository?

Reply:
I got this model from the following [PR](https://github.com/PX4/PX4-Autopilot/pull/21273) opened by Alex and I asked him if we could use it in Discord a while back and he agreed.

Reply:
@PerFrivik But Alex is not the author of this model?